addon/components/o-s-s/attribute-text.ts

```diff
@@ -0,0 +1,32 @@
+import { assert } from '@ember/debug';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+interface OSSAttributeTextArgs {
+  label: string;
+  value?: string;
+  copyable?: boolean;
+  tooltip?: string;
+}
+
+export default class OSSAttributeText extends Component<OSSAttributeTextArgs> {
+  @tracked hasError: boolean = false;
+  @tracked displayInitials: boolean = false;
+
+  constructor(owner: unknown, args: OSSAttributeTextArgs) {
+    super(owner, args);
+  }
+
+  get label(): string {
+    assert(`[component][OSS::Attribute::Text] label is required `, this.args.label);
+    return this.args.label;
+  }
+
+  get value(): string | undefined {
+    return this.args.value === undefined || this.args.value.length === 0 ? '-' : this.args.value;
+  }
+
+  get copyable(): boolean {
+    return !!(this.args.copyable && this.args.value && this.args.value.length > 0);
```

Comment:
```suggestion
    return Bolean(this.args.copyable) && !isBlank(this.args.value)
```
Should do the trick (be sure that the default value for `this.args.copyable` is `true`)